packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/RunnerTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  RunnerTests.swift
+//  RunnerTests
+//
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+
+import XCTest
+
+final class RunnerTests: XCTestCase {
```

Comment:
Are these tests necessary?

From [the wiki](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#lazy-programming).
> Write what you need and no more, but when you write it, do it right.

If the class is necessary in order for the project to compile, then I would remove any code we aren't using.

If you have other plans to implement, then you can ignore this comment.

Reply:
The class itself is necessary, but I could remove all empty function overrides.

Reply:
A small follow up:
I initially setup the RunnerTests target wrong for macOS. Trusting in the automatic generation, I didn't realize that a Swift-Testsuite was newly created and existing (iOS) tests written in Objective-C – tho located in the same directory – were not run on macOS.
This was corrected in https://github.com/flutter/plugins/pull/6517/commits/8865423728d40b4c9a40dc834cfba723ec897a44 and now neither the class nor the file are necessary anymore, hence I deleted the file.